unstructured/partition/text.py

```diff
@@ -52,12 +52,15 @@ def partition_text(
     if filename is not None:
         with open(filename, encoding=encoding) as f:
             try:
-                file_text = f.read()
+                file_text = str(f.read())
```

Comment:
actually I'm surprised here, I thought by opening with the encoding we should be guaranteeing that the file object will always return a string? If that isn't true, would be great to make sure this is specifically caught in a test!

Reply:
I think you're right, this is leftover from tracing down where the error was coming from 😆